cmd/di.go

```diff
@@ -796,7 +794,11 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.Options) (err er
 		TransactorPollTimeout:  options.Payments.RegistryTransactorPollTimeout,
 	}
 
-	if di.IdentityRegistry, err = registry.NewIdentityRegistryContract(di.EtherClientL2, di.AddressProvider, registryStorage, di.EventBus, di.HermesCaller, di.Transactor, registryCfg); err != nil {
+	migrationStorage := migration.NewStorage(di.Storage, di.AddressProvider)
+	registerCallback := func(chainID int64, identity string) {
```

Comment:
I would much rather have the migration subscribe to `AppTopicIdentityRegistration` event and mark the state as needed. This seems like something the registration should not care about.